internal/report.go

```diff
@@ -490,19 +490,19 @@ var IssueDB = map[SchemaIssue]struct {
 	Serial:                {Brief: "Spanner does not support autoincrementing types", severity: warning},
 	AutoIncrement:         {Brief: "Spanner does not support auto_increment attribute", severity: warning},
 	Timestamp:             {Brief: "Spanner timestamp is closer to PostgreSQL timestamptz", severity: suggestion, batch: true},
-	Datetime:              {Brief: "Spanner timestamp is closer to MySQL timestamp", severity: suggestion, batch: true},
-	Time:                  {Brief: "Spanner does not support time/year types", severity: note, batch: true},
-	Widened:               {Brief: "Some columns will consume more storage in Spanner", severity: note, batch: true},
+	Datetime:              {Brief: "Spanner timestamp is closer to MySQL timestamp", severity: warning, batch: true},
```

Comment:
this is a suggestion as marked in: https://docs.google.com/spreadsheets/d/1qx-gS4t0JNki76_bkp-Iag-9cW3MBtfJb2vn7hvm82Y/edit?hl=en&forcehl=1#gid=0